### DISCUSSION TITLE
未能读取数据，因为它的格式不正确。

### DISCUSSION THREAD

**Discussion by GuanWX** [PlayCover/PlayCover#521] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42810500/202194015-de88d2b2-1c54-48cb-9f65-9ce8499abfda.png)

**How should I fix this**

**Comment by deergu** [comment]
I met the same problem, have you solved it?

**Comment by Candygoblen123** [comment, marked as answer]
Hi, please download the 2.0 beta release from our releases page. https://github.com/PlayCover/PlayCover/releases/tag/2.0.0-beta.1
